documentation/text-matching-ranking.html

```diff
@@ -134,12 +135,77 @@ <h3 id="query-trace">Query Trace</h3>
 </p>
 
 
-<!-- ToDo: prefix/substring -->
-<!-- ToDo: streaming search note -->
-<!-- ToDo: refer to
-  https://docs.vespa.ai/documentation/reference/schema-reference.html#match
-  https://docs.vespa.ai/documentation/reference/schema-reference.html#indexing
-  and write some guidance -->
+<h3 id="prefix-search">Prefix Search</h3>
+<p>
+A prefix search will match the query term "hea" to documents with "head".
+However, prefix search is <span style="text-decoration: underline"> not</span>
+enabled in text matching using <em>index</em> mode - some alternatives:
```

Comment:
Consider replacing "enabled" with "supported" as someone might interpret this as it can be turned on.

Reply:
yup